.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "2.7"
+  - "3.6"
```

Comment:
Probably also run on "3.5" since that's the version we use internally, for now at least.